scripts/statesync.bash

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+# microtick and bitcanna contributed significantly here.
+# Pebbledb state sync script.
+set -uxe
+
+# Set Golang environment variables.
+export GOPATH=~/go
+export PATH=$PATH:~/go/bin
+
+# Install Terra with pebbledb 
+go install -ldflags '-w -s -X github.com/cosmos/cosmos-sdk/types.DBBackend=pebbledb' -tags pebbledb ./...
+
+# NOTE: ABOVE YOU CAN USE ALTERNATIVE DATABASES, HERE ARE THE EXACT COMMANDS
+# go install -ldflags '-w -s -X github.com/cosmos/cosmos-sdk/types.DBBackend=rocksdb' -tags rocksdb ./...
+# go install -ldflags '-w -s -X github.com/cosmos/cosmos-sdk/types.DBBackend=badgerdb' -tags badgerdb ./...
+# go install -ldflags '-w -s -X github.com/cosmos/cosmos-sdk/types.DBBackend=boltdb' -tags boltdb ./...
+
+# Initialize chain.
+terrad init test
+
+# Get Genesis
+wget -O ~/.terra/config/genesis.json https://cloudflare-ipfs.com/ipfs/QmZAMcdu85Qr8saFuNpL9VaxVqqLGWNAs72RVFhchL9jWs
+
+# Get "trust_hash" and "trust_height".
+INTERVAL=1000
+LATEST_HEIGHT="$(curl -s https://juno-rpc.polkachu.com/block | jq -r .result.block.header.height)"
+BLOCK_HEIGHT="$((LATEST_HEIGHT-INTERVAL))"
+TRUST_HASH="$(curl -s "https://juno-rpc.polkachu.com/block?height=$BLOCK_HEIGHT" | jq -r .result.block_id.hash)"
+
+# Print out block and transaction hash from which to sync state.
+echo "trust_height: $BLOCK_HEIGHT"
+echo "trust_hash: $TRUST_HASH"
+
+# Export state sync variables.
+export TERRAD_STATESYNC_ENABLE=true
+export TERRAD_P2P_MAX_NUM_OUTBOUND_PEERS=200
+export TERRAD_STATESYNC_RPC_SERVERS="https://rpc-juno-ia.notional.ventures:443,https://juno-rpc.polkachu.com:443"
```

Comment:
`juno` rpcs are used in this PR. Consider updating to `https://terra-rpc.polkachu.com:443`

Reply:
++